backend/src/test/java/com/google/sps/TestEmailNotifier.java

```diff
@@ -0,0 +1,109 @@
+package com.google.sps;
+
+import com.google.sps.notifs.EmailNotifier;
+import org.junit.Assert;
+import org.junit.Test;
+import javax.mail.MessagingException;
+import javax.mail.Session;
+import javax.mail.internet.InternetAddress;
+import javax.mail.internet.MimeMessage;
+import java.io.IOException;
+import java.util.Properties;
+import static org.junit.Assert.*;
```

Comment:
Please switch over to [Truth](www.truth.dev). It's used extensively for Java tests at google and is already used in the other BE code tests.